wgpu-core/src/device/mod.rs

```diff
@@ -195,8 +195,8 @@ pub struct Device<B: hal::Backend> {
     temp_suspected: life::SuspectedResources,
     pub(crate) hal_limits: hal::Limits,
     pub(crate) private_features: PrivateFeatures,
-    limits: wgt::Limits,
-    features: wgt::Features,
+    pub(crate) limits: wgt::Limits,
```

Comment:
are these necessary?

Reply:
this is the last concern, everything else looks gorgeous. Thank you for addressing everything!